cmd/podman/manifest/push.go

```diff
@@ -83,6 +83,10 @@ func init() {
 	flags.StringVar(&manifestPushOpts.SignPassphraseFileCLI, signPassphraseFileFlagName, "", "Read a passphrase for signing an image from `PATH`")
 	_ = pushCmd.RegisterFlagCompletionFunc(signPassphraseFileFlagName, completion.AutocompleteDefault)
 
+	signPassphraseFlagName := "sign-passphrase"
+	flags.StringVar(&manifestPushOpts.SignPassphrase, signPassphraseFlagName, "", "Passphrase to use for signing an image, overrides 'sign-passphrase-file'")
```

Comment:
This option should conflict with --sign-passphrase-file.
If user uses both then throw an error.